scripts/config/facet-upgrade.js

```diff
@@ -3,12 +3,16 @@
  *
  * - addOrUpgrade: list of facets that will be upgraded or added
  * - remove: list of facets that will be completely removed
- * - skip:  mapping "facetName":"listOfFunctionsToBeSkipped". With this you can specify functions that will be ignored during the update.
+ * - skipSelectors:  mapping "facetName":"listOfFunctionsToBeSkipped". With this you can specify functions that will be ignored during the update.
  *          You don't have to specify "initialize()" since it's ignored by default.
  *          Skip does not apply to facets that are completely removed.
+ * - initArgs: if facet initializer expects arguments, provide it here. For no-arg initializers you don't have to specify anything.
+ * - skipInit": list of facets for which you want to skip initialization call.
  */
 exports.Facets = {
-  addOrUpgrade: ["SellerHandlerFacet", "BuyerHandlerFacet"],
+  addOrUpgrade: [],
```

Comment:
I think it makes sense to remove the examples from the actual code, but I think it's a good idea to have an example config in the comments. This is for all of the options.

Reply:
Done.